mvp-spec/website-design.md

```diff
@@ -0,0 +1,130 @@
+# Web site integration: design
+
+## Read data
+
+This diagram details the steps needed to read existing cookies from Prebid SSO
+- at HTTP level **if the website hosts a middleware** that can trigger HTTP redirects when needed
```

Comment:
I wouldn't use the word "middleware". It is a design pattern used in some web frameworks and so it induces internal architecture. What about using semantic at a different level of abstraction?

> - at server level via an HTTP redirect
> - at client level via a Javascript redirect

You can also replace "client" with "browser".

Reply:
I like your phrasing and will use it in the next commit.
However, we need to find **a name for the optional component** that is used at server level to trigger HTTP redirects.
So far, I haven't come up with a better name than "middleware" because it is a component that should be called systematically for any web page requested on the website.
Do you have another suggestion?